build.gradle.kts

```diff
@@ -31,9 +31,11 @@ plugins {
 
 val kotlinVersion: String by project
 val targetJvm: String by project
+val publishingVersion: String by project
 
 subprojects {
-  group = "org.jetbrains"
+  group = "org.jetbrains.projector"
+  version = publishingVersion
```

Comment:
In projector-server, we specify the version a bit differently to projector-client. Please see https://github.com/JetBrains/projector-server/pull/63 with the fix